server/middlewares/csp.js

```diff
@@ -14,7 +14,6 @@ export default function csp() {
   return helmet.contentSecurityPolicy({
     directives: {
       defaultSrc: trusted.concat([
-        '*.optimizely.com',
```

Comment:
you're right, there is a possibility optimizely will also add css and fonts. has been updated:
- `font-src`: because `default-src` is ignored as well for the same reason with `script-src`
- `style-src`: because `default-src` is ignored as well for the same reason with `script-src`
- `default-src`: added for fallback just to be safe but still the key point is `script-src`, `style-src`, `font-src`